### ISSUE TITLE
Freshly installed ubuntu + mailcow "no mailbox selected"

### ISSUE THREAD

**Issue by actionhenkt** [mailcow/mailcow-dockerized#3352]
Please **try** to answer all questions. If you cannot, don't just delete it. If you completely ignore or remove the whole template, **your issue will be closed** with a label "missing information".
Many awesome mailcow supporters don't bother to reply to an issue, when crucial information is missing. Especially when it could have been answered by using this issue template. We need as much information as possible. Please post all logs, do not copy that one line you think may be interesting. Copy the whole bunch and remove sensible information. Thank you!

**Prior to placing the issue, please check following:** *(fill out each checkbox with a `X` once done)*
- [X ] I understand that not following below instructions might result in immediate closing and deletion of my issue.
- [X ] I have understood that answers are voluntary and community-driven, and not commercial support.
- [ ] I have verified that my issue has not been already answered in the past. I also checked previous [issues](https://github.com/mailcow/mailcow-dockerized/issues).

---

**Description of the bug**: What kind of issue have you *exactly* come across?

My issue is...
Fresh install of mailcow is not able to connect to dovecot, shows a note when opening a mailbox "No mailbox selected". 

**Reproduction of said bug**: How *exactly* do you reproduce the bug?

1. installed ubuntu (multiple versions tried, 18.04 / 19,04 / 19.10) all default installs, no firewalls
2. updated installs apt-get update / upgrade
3. installed docker as explained in the documentation : https://mailcow.github.io/mailcow-dockerized-docs/i_u_m_install/
4. installed docker compose
5. set umask to 0022 (from 0002)
6. cloned mailcow from github
7. generated config, config was left default
8. disabled ipv6 in the docker-compose.yml as explained on: https://mailcow.github.io/mailcow-dockerized-docs/firststeps-disable_ipv6/
9. pulled mailcow
10. did a docker-compose up -d
11. logged on with default user/pass
12. set up maildomain
13. made first mailbox
14. opened sogo and logged on to the mailbox
15. message "No mailbox selected"

__I have tried or I do...__ *(fill out each checkbox with a `X` if applicable)*
- [X ] In case of WebUI issue, I have tried clearing the browser cache and the issue persists.
- [ ] I do run mailcow on a Synology, QNAP or any other sort of NAS.

**System information**

Further information (where applicable):

| Question | Answer |
| --- | --- |
| My operating system | ubuntu 18.04 / 19.04 / 19.10 |
| Is Apparmor, SELinux or similar active? | apparmor yes |
| Virtualization technlogy (KVM, VMware, Xen, etc) | esxi |
| Server/VM specifications (Memory, CPU Cores) | 2vcpu / 4g |
| Docker Version (`docker version`) | 19.03.6 |
| Docker-Compose Version (`docker-compose version`) | 1.25.4, build 8d51620a |

Further notes:
 - Output of `git diff origin/master`, any other changes to the code? If so, please post them.
 - All third-party firewalls and custom iptables rules are unsupported. Please check the Docker docs about how to use Docker with your own ruleset. Nevertheless, iptabels output can help _us_ to help _you_: `iptables -L -vn`, `ip6tables -L -vn`, `iptables -L -vn -t nat` and `ip6tables -L -vn -t nat `
 - Check `docker exec -it $(docker ps -qf name=acme-mailcow) dig +short stackoverflow.com @172.22.1.254` (set the IP accordingly, if you changed the internal mailcow network) and `docker exec -it $(docker ps -qf name=acme-mailcow) dig +short stackoverflow.com @1.1.1.1` - output? Timeout?
 
 General logs:
- Please take a look at the [official documentation](https://mailcow.github.io/mailcow-dockerized-docs/debug-logs/).

**Comment by andryyy** [comment]
Which Ubuntu comes with 0002? :/

I install it almost daily on Ubuntu, something fishy is going on here. You can post some logs to the Telegram channel, someone might be able to help you debug this there.

**Comment by actionhenkt** [comment]
user space seems to use 0002, the root user did have 0022. I reinstalled again using the root user.

I dont know what logs you can use, or where to get them but opening the mailbox shows me below logs in the terminal:

`nginx-mailcow_1      | 172.16.25.76 - - [22/Feb/2020:13:25:36 +0100] "GET /SOGo/so/testbox@domain.com/Calendar/alarmslist?browserTime=1582373967 HTTP/2.0" 200 63 "https://172.16.252.17/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.116 Safari/537.36"
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 sogod [69]: 172.16.25.76 "GET /SOGo/so/testbox@domain.com/Calendar/alarmslist?browserTime=1582373967 HTTP/1.0" 200 63/0 0.006 - - 0
nginx-mailcow_1      | 172.16.25.76 - - [22/Feb/2020:13:25:36 +0100] "GET /SOGo.woa/WebServerResources/fonts/MaterialIcons-Regular.woff2 HTTP/2.0" 200 60832 "https://172.16.252.17/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.116 Safari/537.36"
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 sogod [69]: [ERROR] <0x0x5631497ed840[NGImap4ConnectionManager]> IMAP4 login failed:
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 host=172.22.1.250, user=testbox@domain.com, pwd=yes
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 url=imaps://testbox%40domain.com@172.22.1.250/
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 base=(null)
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 base-class=(null))
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 = <0x0x563149ca0d30[NGImap4Client]: login=testbox@domain.com(pwd) address=<0x0x563149716a50[NGInternetSocketAddress]: host=172.22.1.250 port=993>>
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 sogod [69]: <0x563149e24db0[SOGoMailAccount]:0> renewing imap4 password
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 sogod [69]: [ERROR] <0x0x5631497ed840[NGImap4ConnectionManager]> IMAP4 login failed:
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 host=172.22.1.250, user=testbox@domain.com, pwd=yes
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 url=imaps://testbox%40domain.com@172.22.1.250/
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 base=(null)
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 base-class=(null))
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 = <0x0x563149d11460[NGImap4Client]: login=testbox@domain.com(pwd) address=<0x0x563149d5f070[NGInternetSocketAddress]: host=172.22.1.250 port=993>>
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 sogod [69]: [ERROR] <0x563149e24db0[SOGoMailAccount]:0> Could not connect IMAP4
nginx-mailcow_1      | 172.16.25.76 - - [22/Feb/2020:13:25:36 +0100] "GET /SOGo/so/testbox@domain.com/Mail/0/view HTTP/2.0" 200 17 "https://172.16.252.17/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.116 Safari/537.36"
sogo-mailcow_1       | Feb 22 13:25:36 7b6d0f574004 sogod [69]: 172.16.25.76 "GET /SOGo/so/testbox@domain.com/Mail/0/view HTTP/1.0" 200 17/0 0.006 - - 0`


Postfix shows: 

`2/22/2020, 1:22:05 PM	info	daemon started -- version 3.4.7, configuration /opt/postfix/conf
2/22/2020, 1:22:05 PM	info	starting the Postfix mail system
2/22/2020, 1:22:05 PM	warning	warning: symlink leaves directory: /etc/postfix/./makedefs.out
2/22/2020, 1:19:11 PM	warning	warning: /usr/lib/postfix/sbin/smtpd: bad command startup -- throttling
2/22/2020, 1:19:11 PM	warning	warning: process /usr/lib/postfix/sbin/smtpd pid 380 exit status 1
2/22/2020, 1:19:10 PM	crit	fatal: no SASL authentication mechanisms
2/22/2020, 1:19:10 PM	warning	warning: SASL: Connect to inet:dovecot:10001 failed: Connection refused
2/22/2020, 1:19:10 PM	info	connect from mailcowdockerized_sogo-mailcow_1.mailcowdockerized_mailcow-network[172.22.1.248]
2/22/2020, 1:15:52 PM	info	daemon started -- version 3.4.7, configuration /opt/postfix/conf
2/22/2020, 1:15:52 PM	info	starting the Postfix mail system
2/22/2020, 1:15:52 PM	warning	warning: symlink leaves directory: /etc/postfix/./makedefs.out`

**Comment by shiz0** [comment]
I just tried to reproduce this on Ubuntu Server 18.04 LTS, but unfortunately couldn't.
Everything worked as expected.
Closing this, as it seems to be a support case, rather than an issue.
Maybe you missed something during setup? It may help if you observe the logs.
From within /opt/mailcow-dockerized, call:
docker-compose logs -tf --tail="50" | grep "error\|warning\|Error\|Warning\|ERROR\|WARNING"

Also, feel free to join our community support channel on Telegram: https://t.me/mailcow
